### PR TITLE
Improved HTML report template

### PR DESCRIPTION
**Pull request by kunagpal**
#Takes care of #546 and #332 
screen shot 2016-08-23 at 20 51 32

### CODE REVIEW

**Comment by czardoz** [comment]
Can we attach the pass/fail counts to individual assertions rather than a request?

File: lib/reporters/html/index.js [hunk 1]
```diff
@@ -1,18 +1,47 @@
...
+                result[currentExecution.id].failed += assertionStatus.failed || 0;
+            }, {});
+
+        _.forEach(aggregations, function (currentAggregation) {
+            _.set(currentAggregation, 'assertions', assertions[currentAggregation.item.id]);
+        });
+
+        // console.log(JSON.stringify(aggregations, null, 2));
+
+        _.set(this.summary.run, 'aggregations', aggregations);
```
Comment (czardoz) [inline lib/reporters/html/index.js]: This should not be a part of the summary object

Comment (kunagpal) [inline lib/reporters/html/index.js]: Yes, I'll adjust to accommodate individual assertion aggregations as well.

**Comment by kunagpal** [comment]
Done.

**Comment by kunagpal** [comment]
@shamasis Do take a look and let me know if there are any other tasks left on this.

File: lib/reporters/html/index.js [hunk 2]
```diff
@@ -1,22 +1,89 @@
...
+            _.set(currentAggregation, 'assertions', _.values(_.get(assertions, currentId)));
+        });
+
+        _.set(this.summary.run, 'aggregations', aggregations);
+
         this.exports.push({
             name: 'html-reporter',
             default: 'newman-run-report.html',
-            path: options.export,
+            path: options.output,
```
Comment (czardoz) [inline lib/reporters/html/index.js]: The rest of our reporters use `export`. Any reason for this?

Comment (kunagpal) [inline lib/reporters/html/index.js]: `options.export` is no longer being used as a valid key, we had switched to output.

**Comment by czardoz** [comment]
LGTM! @shamasis

**Pull request merged**